packages/core/src/oidc/init.ts

```diff
@@ -125,7 +126,7 @@ export default function initOidc(
             scope: '',
           } satisfies ResourceServer;
 
-          const userId = ctx.oidc.session?.accountId;
+          const userId = oidc.session?.accountId ?? oidc.entities.Account?.accountId;
```

Comment:
@wangsijie please verify.

Reply:
After the update of OIDC, this bug will always return empty on the refresh-token exchange flow.

Reply:
this bug?

Reply:
an existing bug we have.